### ISSUE TITLE
[STORY] Match pipeline users with pipeline developers

### ISSUE THREAD

**Issue by mrusyd** [Australian-Imaging-Service/pipelines#159]
### Links

Epic: #
Feature: #78 

### Description

@mrusyd: I only have a  title for the story on the miro board: "Match-making pipeline users with developers". This may be one or more user and enabler stories, can't be sure. Another story is written that seems to [belong to the same group](https://miro.com/app/board/uXjVOrM3BF4=/?moveToWidget=3458764531549043177&cot=14), possibly offering context:

As a pipeline developer, I would like have a convenient location for viewing feedback, [SO THAT **VALUE**]

**> This story needs to be clarified.**

### Acceptance Criteria

> Acceptance criteria is used to confirm that the story was completed correctly. Criteria can be written in any format, or a "given-when-then" form may be used:

- [ ] 1. [ACCEPTANCE CRITERIA]

### Blocked by

- [ ]
